content/posts/newsletter/2021-02-05.md

```diff
@@ -0,0 +1,235 @@
+---
+title: The XMPP Newsletter January 2021
+save_as: 2021/01/newsletter-01-january/index.html
+url: 2021/01/newsletter-01-january/
+date: 2021-01-05 18:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of December 2020 and January 2021.
+
+Happy to have you back reading! We hope this year will be a great one you, but as well for XMPP!
+
+In December 2020 we discussed the last year of producing and publishing of the XMPP Newsletter, [read the notes here](https://github.com/xsf/xmpp.org/pull/851#issuecomment-754919811)!
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the last year, please consider to say thanks or help these projects!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- French on [linuxfr.org](https://linuxfr.org/tags/xmpp/public) and [jabberfr.org](https://news/jabberfr.org/category/newsletter/).
+
+## XSF Announcement
+
+Regarding the announcement by Let's Encrypt to switch away from their Root CA certificate: Now they are extending their Android device compatibility of their certificates](https://letsencrypt.org/2020/12/21/extending-android-compatibility.html). [Read their the orginial post](https://letsencrypt.org/2020/11/06/own-two-feet.html). 
+
+Edward Maurer wrote a commentary blog post regarding on the current situation of makeing a choice about the choice of the way to communicate: [Instant Messaging: It's not about the app](https://xmpp.org/2021/01/instant-messaging-its-not-about-the-app/). There are also translations in [German](https://xmpp.org/2021/01/instant-messaging-es-geht-nicht-um-die-app/), [French](https://xmpp.org/2021/01/messagerie-instantanee-il-ne-sagit-pas-de-lapplication/), [Spanish](https://xmpp.org/2021/01/mensajeria-instantanea-no-se-trata-de-la-aplicacion/) and [Romanian](https://xmpp.org/2021/01/mesagerie-instantanee-nu-este-vorba-despre-aplicatie/). Thanks to all contributors!
```

Comment:
"making a choice about the way we communicate" maybe (spelling on making and rephrasing the sentence because I didn't understand what it was trying to say before)

Reply:
Better?